linkerd/app/integration/src/identity.rs

```diff
@@ -217,7 +216,7 @@ impl Controller {
 
 impl pb::server::Identity for Controller {
     type CertifyFuture =
-        Box<dyn Future<Item = grpc::Response<pb::CertifyResponse>, Error = grpc::Status> + Send>;
+        Box<dyn Future01<Item = grpc::Response<pb::CertifyResponse>, Error = grpc::Status> + Send>;
```

Comment:
Is this because we haven't moved to tonic yet?

Reply:
Yup, that's right.